btcec/schnorr/musig2/sign.go

```diff
@@ -320,6 +332,19 @@ func Sign(secNonce [SecNonceSize]byte, privKey *btcec.PrivateKey,
 		return nil, err
 	}
 
+	nonce.ToAffine()
+
+	var adaptedNonce *btcec.JacobianPoint
```

Comment:
How about `adaptedNonce := nonce` to avoid `else` case? Same for `verifyPartialSig`.